.github/workflows/ci.yml

```diff
@@ -75,25 +75,35 @@ jobs:
   unit-tests:
     name: Unit tests
     needs: [pre-commit, towncrier, package]
+    runs-on: ${{ matrix.platform }}-latest
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         platform: ["macos", "ubuntu", "windows"]
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
+        use-win-store-python: [ false ]
```

Comment:
Looking at an alternate way to structure this - rather than adding `use-win-store-python` to the build matrix, could we add `windows-winstore` as a platform (excluding 3.8, 3.9 etc from the matrix) The setup you have here works, but it renders the target platforms as "macOS, 3.8, false", with the winstore build being "windows, 3.11, true, false".

Reply:
Agreed; I've been doing a lot of fighting with github's matrix configuration....and landed here for testing. Using a new platform does make the most sense....but we're driving the runner OS off of it and string manipulation isn't really supported in the yaml. I'll see what I can do.

Reply:
The absence of string manipulation support in github action expressions is bit annoying. But I found actions/runner#409 and it helped arbitrate the `runs-on` value. Open to critique.

```
runs-on: ${{ fromJSON('{"winstore":"windows"}')[matrix.platform] || matrix.platform }}-latest
```